stratum/CMakeLists.txt

```diff
@@ -348,58 +331,110 @@ target_link_libraries(stratum PUBLIC
     p4v1_proto
 )
 
+target_link_libraries(stratum PUBLIC glog gflags grpc protobuf grpc++)
+
+target_link_libraries(stratum PUBLIC
+    absl::strings absl::synchronization absl::graphcycles_internal
+    absl::stacktrace absl::symbolize absl::malloc_internal
+    absl::debugging_internal absl::demangle_internal absl::time
+    absl::strings_internal absl::throw_delegate
+    absl::base absl::spinlock_wait absl::int128 absl::raw_logging_internal
+    absl::log_severity absl::civil_time absl::civil_time absl::time_zone
+    absl::status absl::cord absl::cord_internal absl::cordz_info
+    absl::cordz_handle absl::cordz_sample_token absl::cordz_functions
+    absl::exponential_biased absl::str_format_internal absl::hash
+    absl::raw_hash_set absl::city absl::bad_optional_access
+    absl::bad_variant_access absl::low_level_hash
+)
+
+target_link_libraries(stratum PUBLIC pthread)
+
```

Comment:
The long-term plan should be to create CMake package files for the DPDK and Tofino SDEs that would encapsulate finding the libraries and creating targets for them. Then we could simply do a find_package() to obtain the definitions.